src/components/userCard.tsx

```diff
@@ -0,0 +1,27 @@
+import projectsCardIcon from "../assets/icons/projectsCardIcon.svg";
```

Comment:
Suggestion: change for absolute path

Reply:
yep i forgot about that, thanks for the heads up